components/authorization/cluster/stable-syncer-namespace-admin.yaml

```
@@ -0,0 +1,12 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: kcp-syncer-appstudio-internal-209j7z8n-admin
+  namespace: kcp-syncer-appstudio-internal-209j7z8n
+subjects:
+  - kind: User
+    name: skabashnyuk
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: admin
```

it's just a detail, but `edit` should be enough
```suggestion
  name: edit
```